bokehjs/test/models/tools/edit/box_edit_tool.ts

```diff
@@ -79,6 +79,17 @@ const make_testcase = function(): BoxEditTestCase {
 
 describe("BoxEditTool", () =>
 
+  describe("Model", function(): void {
+
+    it("should create proper tooltip", function(): void {
+      const tool = new BoxEditTool({})
+      expect(tool.tooltip).to.be.equal('Box Edit Tool')
+
+      const custom_tool = new BoxEditTool({custom_tooltip: 'Box Edit Custom'})
+      expect(custom_tool.tooltip).to.be.equal('Box Edit Custom')
+	});
```

Comment:
Bad indentation (same in other test files). Looks like a tab (?). I've been seeing this in other places in recent commits.

Reply:
Thanks, missed this one, I messed up my emacs config and haven't had time to fix it.